pxr/usdImaging/usdImagingGL/drawModeAdapter.cpp

```diff
@@ -322,11 +338,13 @@ UsdImagingGLDrawModeAdapter::_ComputeGeometryData(
     VtValue* assign) const
 {
     if (drawMode == UsdGeomTokens->origin) {
-        *extent = _ComputeExtent(prim);
+        *extent = _ComputeExtent(prim,
+            _HasVaryingExtent(prim) ? time : UsdTimeCode::EarliestTime());
```

Comment:
A small thing, but it seems less error-prone to have the _HasVaryingExtent inside the _ComputeExtent call (and maybe put a warning comment in the header?).

Reply:
I did consider this, but I didn't like the semantics of calling "_ComputeExtent(time)" and having it return an extent that was computed at some other time than what the caller asked for. Still happy to make the change because they are obviously functionally equivelent, but wanted to explain my reasoning before proceeding.

Reply:
That's fair, I'm ok with skipping that change.